tests/sles4sap/saptune/mr_test_run.pm

```diff
@@ -34,8 +34,12 @@ our $result;
 our $result_module;
 
 sub reboot_wait {
-    my ($self) = @_;
-    $self->reboot;
+    # Do not reboot if testing in public cloud instance,
```

Comment:
rebooting in clouds is tough and complicated you right , but if you need this reboot and if you doing it in other tests does it really make sense simply ignore this ? Is your test still valid if you not doing reboot ? In some cases (e.g. kernel update)  reboot is vital and without it further testing does not make any sense . While reboot is complex and error prone for public cloud tests it is not impossible

Reply:
I have asked in team no concrete answer yet, if it is essential I will open another JIRA ticket to track this comment.
Thank you so much for raising it up.